.github/workflows/build.yml

```diff
@@ -56,6 +79,7 @@ jobs:
         run: cargo check --package atsamx7x-hal --features ${{ matrix.features }},samv71q21b,unproven
 
   board-examples:
+    if: github.event.pull_request.head.repo.full_name != github.repository
```

Comment:
@michalfita What is the purpose of this line?

Reply:
To stop redundant builds. We have two triggers `push` and `pull_request`: the former means we build everything we push **including** in PRs, the latter means changes in the PR are build, including branches in forks (outside contributors). But having both stupidly doubles each job. This condition let jobs from `pull_request` trigger on forks to be run, but stops on our own branches. Unfortunately, this condition cannot exists at higher level, has to be per job 🤦🏼‍♂️.